phone/app/src/main/res/values/colors.xml

```diff
@@ -7,4 +7,5 @@
     <color name="teal_700">#FF018786</color>
```

Comment:
<table data-meta="generated_by_danger">
  <tbody>
    <tr>
      <td>:warning:</td>
      <td width="100%" data-sticky="false"><span data-href="https://github.com/mayoi-design/maigo_compass_android/blob/21755dd43e1fd8fb220ebe47cc8c52b679645322/phone/app/src/main/res/values/colors.xml#L7"></span>The resource <code>R.color.teal_700</code> appears to be unused</td>
    </tr>
  </tbody>
</table>